commands/logs_e2e_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/mattermost/mmctl/client"
+
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlE2ETestSuite) TestlogsCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForAllClients("Display single log line", func(c client.Client) {
+		cmd := &cobra.Command{}
+		cmd.Flags().Int("number", 1, "")
+
+		data, err := testLogsCmdF(s.th.SystemAdminClient, cmd, []string{})
```

Comment:
We should use the `client` from the function parameter.

```suggestion
		data, err := testLogsCmdF(c, cmd, []string{})
```